### PR TITLE
SN-26/Create Like model

### PR DESCRIPTION
**Pull request by skripchenkoVeta**

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#  likable_id   :bigint
+#  user_id      :bigint
+#
+# Indexes
+#
+#  index_likes_on_likable  (likable_type,likable_id)
+#  index_likes_on_user_id  (user_id)
+#
+class Like < ApplicationRecord
+  belongs_to :likable, polymorphic: true
```
Comment (mlshvch) [inline app/models/like.rb]: `  belongs_to :likeable, polymorphic: true`

**Review by PivovarAndrew — APPROVED** [review]

File: app/models/like.rb (hunk 1 above)
Comment (mlshvch) [reply app/models/like.rb]: I suggest this variant, such as 'imageable in this guide'
https://guides.rubyonrails.org/association_basics.html#polymorphic-associations

File: db/migrate/20220123184843_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# CreateLikes migration
+class CreateLikes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :likes do |t|
+      t.references :user
+      t.references :likable, polymorphic: true
```
Comment (mlshvch) [inline db/migrate/20220123184843_create_likes.rb]: ditto

File: spec/factories/likes.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Indexes
+#
+#  index_likes_on_likable  (likable_type,likable_id)
+#  index_likes_on_user_id  (user_id)
+#
+FactoryBot.define do
+  factory :like do
+    association :user
+    association :likable, factory: :post
```
Comment (mlshvch) [inline spec/factories/likes.rb]: ditto

**Review by mlshvch — APPROVED** [review]

**Review by totsamyroma — APPROVED** [review]

**Pull request merged**